SIGMA/src/main/kotlin/sigma/dataAccess/impl/parsers/CsvTimelineParser.kt

```diff
@@ -0,0 +1,77 @@
+package sigma.dataAccess.impl.parsers
+
+import sigma.dataAccess.impl.data.Timeline
+import sigma.dataAccess.impl.data.CompletionStatus
+import sigma.dataAccess.impl.data.Day
+import sigma.dataAccess.model.parsers.ITimelineParser
+import sigma.dataAccess.model.loggers.ILogger
+import java.io.File
+import java.time.LocalDate
+
+class CsvTimelineParser(private val logger: ILogger) : ITimelineParser {
```

Comment:
nie jestem zwolennikiem pisania własnych parserów formatów takich jak csv, bo to żmudna robota, a i tak nie zrobi się tego lepiej niż istniejąca biblioteka. A chyba jeszcze lepiej użyć jsona i kotlinx.serialization